services/app-web/src/contexts/AuthContext.tsx

```diff
@@ -139,6 +174,9 @@ function AuthProvider({
                 loginStatus,
                 logout,
                 checkAuth,
+                isSessionExpiring,
```

Comment:
We’ve got enough stuff going on in here that I think we should do a little documentation. Could you write up what these three new varables do and when they are set? For instance, it looks like timeUntilLogout isn’t set until we are displaying the alert. 

nit: I think sessionIsExpiring would be slightly more clear, isSessionExpiring sounds a little like a question to me for some reason.